client/src/components/Workflow/WorkflowIndexActions.vue

```diff
@@ -32,10 +32,10 @@ export default {
     },
     methods: {
         navigateToCreate: function () {
-            window.location.assign(`${this.root}workflows/create`);
+            this.$router.push(`${this.root}workflows/create`);
```

Comment:
Thank you for fixing this. No need to parse the `root` prop here which can actually be removed from the component. Writing: `this.$router.push("/workflows/create"))` is sufficient. Thanks again.

Reply:
ok. I will change that and push it again. The unit test is also failing because they are old, we have to change the unit tests as well.